spec/models/local_authority/postcode_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: local_authority_postcodes
+#
+#  gss_code   :string           not null
+#  value      :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_local_authority_postcodes_on_gss_code  (gss_code)
+#  index_local_authority_postcodes_on_value     (value) UNIQUE
+#
+describe LocalAuthority::Postcode, type: :model do
```

Comment:
```suggestion
describe LocalAuthority::Postcode do
```

This is not needed as it's inferred by RSpec based on the directory.